diff.js

```diff
@@ -39,18 +39,20 @@ import { equalityStrict } from './function.js'
  * @return {SimpleDiff<string>} The diff description.
  */
 export const simpleDiffString = (a, b) => {
+  const aArr = Array.from(a)
+  const bArr = Array.from(b)
```

Comment:
_If_ these computations below to prove to be less performant then perhaps we could compare the length of the array to the original input string and fallback to the original simpler string modifications when possible.

Reply:
Good job!